rest_framework/pagination.py

```diff
@@ -715,6 +708,19 @@ def get_ordering(self, request, queryset, view):
             return (ordering,)
         return tuple(ordering)
 
+    def decode_cursor(self, request):
+        # Determine if we have a cursor, and if so then decode it.
+        encoded = request.query_params.get(self.cursor_query_param)
+        if encoded is not None:
```

Comment:
I tend to find it neater in methods like this to prefer the less indented style, so...

    encoded = request.query_params.get(self.cursor_query_param)
    if encoded is None:
        return   # No cursor query parameter was included

      # ... remainder of method